crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -55,6 +59,11 @@ def checks
             passed: clusters_healthy?,
             errors: clusters_health_check_errors
           } if Api::Crowbar.addons.include?("ha")
+
+          return ret unless upgrade_status.current_step == "upgrade_prechecks"
```

Comment:
Here you are checking for strings, and in other places for symbols. We need to check for symbols everywhere (esp. after https://github.com/crowbar/crowbar-core/pull/769)